web/src/views/IndividualResults/StudyPage.vue

```diff
@@ -238,27 +238,28 @@ export default defineComponent({
           cols="12"
           md="7"
         >
-          <IndividualTitle :item="item">
-            <template #default>
-              <div v-if="item.omics_processing_counts">
-                <template v-for="val in item.omics_processing_counts">
-                  <v-chip
-                    v-if="val.count && (val.type.toLowerCase() !== 'lipidomics')"
-                    :key="val.type"
-                    small
-                    class="mr-2 my-1"
-                    @click="seeOmicsForStudy(val.type)"
-                  >
-                    {{ fieldDisplayName(val.type) }}: {{ val.count }}
-                  </v-chip>
-                </template>
-              </div>
-            </template>
-          </IndividualTitle>
-          <InvestigatorBio
-            v-if="item.principal_investigator"
-            :item="item"
-          />
+          <v-container>
+            <IndividualTitle :item="item">
+              <template #default>
+                <div v-if="item.omics_processing_counts">
+                  <template v-for="val in item.omics_processing_counts">
+                    <v-chip
+                      v-if="val.count && (val.type.toLowerCase() !== 'lipidomics')"
+                      :key="val.type"
+                      small
+                      class="mr-2 my-1"
+                      @click="seeOmicsForStudy(val.type)"
+                    >
+                      {{ fieldDisplayName(val.type) }}: {{ val.count }}
+                    </v-chip>
+                  </template>
+                </div>
+              </template>
+            </IndividualTitle>
+            <InvestigatorBio
+              :item="item"
+            />
```

Comment:
bf85bb0c449f8504663b892b86bab70698daa355 I changed it to `TeamInfo` since there is now Study and Consortia so I figured it was best to keep it generic